package.json

```diff
@@ -18,7 +18,10 @@
   "scripts": {
     "dev": "turbo run dev",
     "build": "turbo run build",
-    "format": "turbo run format",
+    "lint:all": "turbo run lint",
+    "lint": "prettier . --check",
```

Comment:
Do we actually need top-level `format` and `lint` scripts? For the `README.md` and `package.json`? I would prefer having clearer scripts/setup by ditching those and only having the turbo-commands for the actual packages. But as you like.

Reply:
I included a root `lint` because this not only covers files in `/` but also e.g. github workflows. And our `deploy.yml` workflow file actually didn't follow our prettier rules. There might follow more files in our monorepo root, e.g. some scripts. So I'd prefer to leave a top-level `lint` in. IMO it's not consistent to require all files in packages to have a certain format (prettier, eslint, ...) but then not require it in files outside packages.

But I'm happy to change the name of scripts or the setup somehow if this is confusing. Just didn't see a clearer way of doing it while avoiding name clashes/conflicts for turbo.

Reply:
Ok! Then let's keep them. I would now either rename the `frontend:…` or the `lint` tasks to be consistent about where the task-scope is placed (e.g. plain tasks are executing the turbo pipes). Maybe:

* `dev: turbo run dev`
* `lint: turbo run lint`
* `root:lint prettier . --check`
* `frontend:dev pnpm --filter frontend dev`

Or we ditch plain dev/build tasks and do it like:

* `dev:all turbo run dev`
* `lint:all turbo run lint`
* `lint prettier . --check`
* `dev:frontend pnpm --filter frontend dev`

I prefer the first. But I can also live with the status quo tbh :)

Reply:
Because of the way turbo resolves nodejs tasks, they have to be named the same in all packages. So all package-scoped `lint` tasks have to be called the same, that is, `lint`. This is the reason, the repo-wide, i.e., multi-package, lint script name has to be different from `lint` in the `root` package or we cannot run the root package's `lint` with turbo. Hence I named it `lint:all`.

Regarding `frontend:dev` and `frontend:start`, I also wanted to suggest to change the naming convention here to name such tasks first by the type and then the scope, i.e. `dev:frontend` and `start:frontend`. But didn't want to bloat this PR. Happy to do it now.

Reply:
gotcha. yes, let's do the `frontend:x` -> `x:frontend` swap :)

Reply:
done